torch/_inductor/config.py

```diff
@@ -770,7 +770,7 @@ class triton:
     autotune_cublasLt = True
 
     # Tune the generated Triton kernels at compile time instead of first time they run
-    autotune_at_compile_time = False
+    autotune_at_compile_time = True
```

Comment:
@eellison  is right. This is not meant default to True, otherwise, training will be affected.